src/Reducer.js

```diff
@@ -72,7 +69,8 @@ function update(state,action){
             if (el.children.length > 1) {
                 el.children.pop();
                 el.index = el.children.length - 1;
-                newState.scenes.current = getCurrent(newState);
+                delete newState.scenes[newState.scenes.current];
```

Comment:
Deleting looks not good - scenes should store all possible scenes, like library.